docker-compose.zappdev.yaml

```diff
@@ -36,9 +36,10 @@ services:
       - shared_contracts_zappdev:/build:rw
 
   point_node:
-    extends:
-      file: docker-compose.base.yaml
-      service: point_node
+    build:
```

Comment:
This will slow down the `zappdev` env, what's the advantage of this change?

Reply:
I just copied it from the `test` compose file, as with the original config it was starting outdated point node image. However, I'll roll it back so guys from Darren's team can decide themselves

Reply:
> I just copied it from the `test` compose file, as with the original config it was starting outdated point node image. However, I'll roll it back so guys from Darren's team can decide themselves

Yeah, it makes sense to build the image in the test compose because we always want to test it against the latest changes. But it case of `zappdev` compose it is a bit different, we need to focus of the `zapps` development, so re-building the image will probably be very annoying.